web/src/js/index.js

```diff
@@ -374,12 +374,18 @@ function getAllowlist() {
       dayjs.extend(dayjs_plugin_timezone);
 
       app.allowlist = allowlistRaw.map((item) => {
+        // Parse Resource ID, i.e. split on ":"
+        let parsedResourceId = item["resource_id"].split(":", 2);
+        parsedResourceId.push(
+          item["resource_id"].slice(parsedResourceId.join("").length + 2)
+        );

```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/servian/aws-auto-cleanup/pullRequest?prid=9792602)
```suggestion
        );
```